packages/react-native-sdk/src/components/Call/CallLayout/CallParticipantsGrid.tsx

```diff
@@ -57,25 +50,35 @@ export const CallParticipantsGrid = ({
   const isUserAloneInCall = remoteParticipants?.length === 0;
   const participants = showFloatingView ? remoteParticipants : allParticipants;
 
-  if (showFloatingView && isUserAloneInCall) {
-    return <LocalParticipantView layout={'fullscreen'} />;
+  if (isUserAloneInCall) {
+    return (
+      <>
+        {LocalParticipantView && <LocalParticipantView layout={'fullscreen'} />}
+      </>
+    );
   }
```

Comment:
inteersting what this will do is..

if integrator doesn't want a floating view so passes null here..

but when he is alone he will not see anything

Reply:
Does other SDKs have a full screen local video component?

Reply:
the whole local participnat view component seems to have no customisation support right?

Reply:
I have removed the fullscreen mode all in all from LocalParticipantView